spec/models/user_spec.rb

```diff
@@ -3,39 +3,61 @@
 RSpec.describe User, type: :model do
   describe 'Factory' do
     it 'has a valid factory' do
-      # Testear que el factory definido es válido.
+      build(:user).should be_valid
```

Comment:
Acá también cambiaría por `build_stubbed` y la forma con `expect` en lugar de `should`.